examples/demo_interface.pp

```diff
@@ -56,4 +56,10 @@
     duplex      => 'auto',
     speed       => '100m',
   }
+  #  Requires F3 or newer linecards
```

Comment:
Not blocking but eventually I think we will want to use facter facts to set these property values to 'undef' if not supported so that we don't have to leave this commented out.